it/controllers/pensions/incomeFromPensions/RemovePensionSchemeControllerISpec.scala

```diff
@@ -0,0 +1,211 @@
+/*
+ * Copyright 2022 HM Revenue & Customs
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+package controllers.pensions.incomeFromPensions
+
+import builders.PensionsUserDataBuilder.aPensionsUserData
+import builders.UserBuilder.aUserRequest
+import org.jsoup.Jsoup
+import org.jsoup.nodes.Document
+import org.scalatest.BeforeAndAfterEach
+import play.api.http.HeaderNames
+import play.api.http.Status.{BAD_REQUEST, OK, SEE_OTHER}
+import play.api.libs.ws.WSResponse
+import utils.PageUrls.{fullUrl, pensionSummaryUrl}
+import utils.PageUrls.IncomeFromPensionsPages.{removePensionSchemeUrl, ukPensionSchemeSummaryListUrl}
+import utils.{IntegrationTest, PensionsDatabaseHelper, ViewHelpers}
+
+class RemovePensionSchemeControllerISpec extends IntegrationTest with ViewHelpers with BeforeAndAfterEach with PensionsDatabaseHelper {
+
+  object Selectors {
+    val captionSelector: String = "#main-content > div > div > form > header > p"
+    val cancelLinkSelector: String = "#cancel-link-id"
+  }
+
+  trait CommonExpectedResults {
+    val expectedTitle: String
+    val expectedHeading: String
+    val expectedCaption: Int => String
+    val buttonText: String
+    val cancelText: String
+  }
+
+  object CommonExpectedEN extends CommonExpectedResults {
+    val expectedTitle = "Are you sure you want to remove pension name 1?"
```

Comment:
It might be good to use a val (e.g. from anUkPensionIncomeViewModelOne) rather than a hardcoded pension scheme name? In case it ever changes the tests wont break